sammba/data_fetchers/tests/test_atlas.py

```diff
@@ -162,3 +162,27 @@ def test_fetch_atlas_waxholm_rat_2014():
                               (.2, .2, .2))
     assert_array_almost_equal(nibabel.load(bunch['maps']).header.get_zooms(),
                               (.2, .2, .2))
+
+
+@with_setup(setup_mock, teardown_mock)
+@with_setup(tst.setup_tmpdata, tst.teardown_tmpdata)
+def test_fetch_atlas_lemur_mircen_2017():
+    datadir = os.path.join(tst.tmpdir, 'mircen_2017')
+    os.mkdir(datadir)
+    dummy = open(os.path.join(
```

Comment:
not sure what this is for; the real text file is available on the server somewhere. even if needed, as stated above, should be done with pandas

Reply:
This is for test. No download is done and the dummy file is created to be parsed by the test, and then it is removed.